lib/ReactViews/WelcomeMessage/WelcomeMessage.jsx

```diff
@@ -87,29 +131,161 @@ export const WelcomeMessagePure = props => {
         onExited: () => {
           if (shouldExploreData) {
             setShouldExploreData(false);
-            runInAction(() => {
-              viewState.topElement = "AddData";
-              viewState.openAddData();
-            });
+            viewState.openAddData();
+            viewState.setTopElement("AddData");
+          }
+          if (shouldOpenHelp) {
+            setShouldOpenHelp(false);
+            viewState.showHelpPanel();
           }
         }
       }}
     >
-      <div
-        className={classNames({
-          [Styles.welcomeModalWrapper]: true,
-          "top-element": isTopElement
-        })}
-        onClick={handleClose.bind(null, false)}
+      <WelcomeModalWrapper
+        fullWidth
+        fullHeight
+        positionAbsolute
+        right
+        onClick={() => handleClose(false)}
       >
-        <SlideUpFadeIn isVisible={welcomeVisible}>
-          <article
+        <Box
+          styledWidth={
+            viewState.isMapFullScreen || viewState.useSmallScreenInterface
+              ? "100%"
+              : "calc(100% - 350px)"
+          } // TODO: use variable $work-bench-width
+          fullHeight
+          centered
+        >
+          <VideoGuide
+            viewState={viewState}
+            videoLink={"https://www.youtube.com/embed/NTtSM70rIvI"}
+            background={
+              "https://img.youtube.com/vi/NTtSM70rIvI/maxresdefault.jpg"
+            }
+            videoName={WELCOME_MESSAGE_VIDEO}
+          />
+          <SlideUpFadeIn isVisible={welcomeVisible}>
+            <Box
+              styledWidth={"667px"}
+              styledHeight={"504px"}
```

Comment:
Another one - better to make this a styledMinHeight instead (you'll need to update Box to take that prop) - usually if you want to set heights for some spacing reasons, lean towards min height as you'll have things reflowing. In this instance, if you resize the window close enough, the "don't show me this again" will appear outside of the box (but y ou can't see it unless you look at the browser layout of the box due to the transparent bg.
<img width="514" alt="image" src="https://user-images.githubusercontent.com/9134432/81767434-c5002c80-951b-11ea-94a7-e200ffe13dab.png">

Reply:
(& if you'll please do the same for Button.jsx where I haven't done this exact reasoning 😆 🙏 )